addons/crm/views/res_partner_views.xml

```diff
@@ -40,4 +40,39 @@
             </field>
         </record>
 
+        <!-- Identical to base.action_partner_form, but adds different action helper for CRM -->
```

Comment:
I don't understand purpose of this action compared to  base.action_partner_form

Reply:
This one is just for the different action helper that we need specifically in CRM.

Reply:
If this is just to change "new customer" to "Contact" I think this is adding views and code for nothing. We could change base action, or just skip it... Better have less actions, easier for customizing.